### PR TITLE
Language reworked

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: docs/admin/projects.rst [hunk 1]
```diff
@@ -384,36 +379,36 @@ Check resource limits
 +++++++++++++++++++++
 
 If you are importing huge translations or repositories, you might be hit by
-resource limits of your server.
+resource limitations of your server.
 
-* check amount of free memory, having translation files by system will greatly improve performance
-* disk operations might be bottleneck if there is lot of strings to process - the disk is pushed by both Weblate and database
-* additional CPU cores might help to improve performance of background tasks (see :ref:`celery`)
+* Check the amount of free memory, having translation files by the system memory will greatly improve performance.
```
Comment (comradekingu) [inline docs/admin/projects.rst]: Is this correct now?

File: docs/admin/projects.rst [hunk 2]
```diff
@@ -115,104 +113,102 @@ Access control
...
-    It is recommended to have translation components of reasonable size - split
-    the translation by anything what makes sense in your case (individual
-    applications or addons, book chapters or websites).
+    It is recommended to keep translation components to a reasonable size - split
+    the translation by anything that makes sense in your case (individual
+    apps or addons, book chapters or websites).
 
-    Weblate easily handles translations with 10000 of strings, but it is harder
-    to split work and coordinate among translators with such a large translation.
-    Also when one translator is working on a component, this translation is locked
```
Comment (comradekingu) [inline docs/admin/projects.rst]: Components allow concurrent users now, right?

Comment (nijel) [reply docs/admin/projects.rst]: Yes, thanks for spotting.

File: docs/admin/projects.rst (hunk 1 above)
Comment (nijel) [reply docs/admin/projects.rst]: the original sentence probably should be "having translation files cached by system"

Comment (nijel) [reply docs/admin/projects.rst]: I've clarified this in 9916ca79f1.

**Comment by nijel** [comment]
Merged, thanks for your contribution!

**Pull request merged**